src/main/scala/com/github/swagger/akka/SwaggerHttpService.scala

```diff
@@ -33,24 +32,14 @@ object SwaggerHttpService {
   val logger = LoggerFactory.getLogger(classOf[SwaggerHttpService])
   val readerConfig = new DefaultReaderConfig
 
-  def toJavaTypeSet(apiTypes: Seq[Type]): Set[Class[_]] = {
-    apiTypes.map(t => getClassForType(t)).toSet
-  }
-
-  private lazy val mirror = scala.reflect.runtime.universe.runtimeMirror(getClass.getClassLoader)
-
-  def getClassForType(t: Type): Class[_] = {
-    mirror.runtimeClass(t.typeSymbol.asClass)
-  }
-
   def removeInitialSlashIfNecessary(path: String): String =
     if(path.startsWith("/")) removeInitialSlashIfNecessary(path.substring(1)) else path
 }
 
 trait SwaggerHttpService extends Directives {
 
   import SwaggerHttpService._
-  val apiTypes: Seq[Type]
+  def apiClasses: Set[Class[_]]
```

Comment:
I also suggest using defs in other places also, because it is more flexible and can be overriden by vals as well as vars